### PR TITLE
added: AuthHOC for Authorization render component

### PR DESCRIPTION
**Pull request by r17x**
@evilfactorylabs/nadya Maybe solve issue https://github.com/evilfactorylabs/nadya/issues/23

### CODE REVIEW

File: app/src/fragments/Shell.js [hunk 1]
```diff
@@ -23,26 +21,14 @@ export default class App extends Component {
...
-  }
   _handleDialogOpen = () => {
     this.setState({ isDialogOpen: true })
   }
   _handleDialogClose = () => {
     this.setState({ isDialogOpen: false })
   }
   render() {
-    return (
+    return AuthHOC(() => (
```
Comment (r17x) [inline app/src/fragments/Shell.js:31]: I Never expect before, for this implementation.

**Comment by anakmagang** [comment]
Current deployment URL: https://nadya-app-larlm1mq7.now.sh

File: app/@nadya:v1/LOG [hunk 2]
```diff
@@ -0,0 +1,3 @@
+2019/06/26-18:47:03.110900 70000f678000 Recovering log #15
```
Comment (faultables) [inline app/@nadya:v1/LOG]: I never expect this files before. where did they came?

Comment (r17x) [reply app/@nadya:v1/LOG]: I remove directory , that’s directory has created if multiple git remote on the same url 

origin: xxx.git
upstream: xxx.git 

I think, I’m still working on forked repo but actually in same repository. :sorry for my bad

**Comment by anakmagang** [comment]
Current deployment URL: https://nadya-app-gxlhncilr.now.sh

File: app/@nadya:v1/LOG (hunk 2 above)
Comment (faultables) [reply app/@nadya:v1/LOG]: Yeaa no problem

**Review by kevanantha — APPROVED** [review]

**Review by faultables — APPROVED** [review]
Thanks for the PR!

Basically LGTM, so I approve it

**Pull request merged**